appserver/packager/external/jakarta-ee11-shim/pom.xml

```diff
@@ -227,7 +236,12 @@
                                     jakarta.ws.rs.container;
                                     jakarta.ws.rs.core;
                                     jakarta.ws.rs.ext;
-                                    jakarta.ws.rs.sse;version="3.1.99";shim=true
+                                    jakarta.ws.rs.sse;version="3.1.99";shim=true,
+
+                                    <!-- Downgrade shim for JSP, required by WASP. Remove once WASP upgraded -->
```

Comment:
This comment isn't accurate - this PR _is_ upgrading WASP.
It's the JSTL API which isn't resolving.

Reply:
That explains why I couldn't remove the shim 😆 Updated.